Sources/InternxtSwiftCore/Services/Network/Upload.swift

```diff
@@ -31,5 +32,7 @@ struct Upload {
         let iv = cryptoUtils.hexStringToBytes(String(hexIv))
         
         let fileKey = try encrypt.generateFileKey(mnemonic: mnemonic, bucketId: bucketId, index: index!)
+        
+        // File is encrypted at outputFilePath, make sure by reverse verifying the content hash
```

Comment:
You may match the hash calculated at encryption and also other metadata like the size, so if the encryption still fails for some reason, you still spot the issue indirectly

Reply:
Good point, I'll add it to the next PR since this is still missing some parts